src/app/clothes/page.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+import { useState, useEffect } from 'react';
+import { Button } from '@/components/ButtonStyle';
+import { getCategory } from '@/functions/getCategory';
+import { getData } from '@/functions/getData';
+import { normalizeImage } from '@/functions/normalizeImage';
+import { stringifyUrl } from '@/functions/stringifyUrl';
+import { ItemClothes } from '@/types/itemClothes';
+import { Card, CardFooter, Image } from '@nextui-org/react';
+import Link from 'next/link';
+import React from 'react';
+import { dataClothes } from '@/mocks';
+
+const Page = () => {
+  const [data, setData] = useState<ItemClothes[]>([]);
+  const [selectedCategory, setSelectedCategory] = useState<string>('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const fetchedData: ItemClothes[] = await getData();
+      setData(fetchedData);
+    };
+
+    fetchData();
+  }, []);
+
+  const filteredData = selectedCategory
+    ? data.filter(item => item.category === selectedCategory)
+    : data;
+
+  return (
+    <div className="flex min-h-[80vh] flex-col items-center justify-between gap-4">
+      <div className="flex flex-row flex-wrap items-center gap-4 px-8 justify-center">
+        {dataClothes.filters.map(filter => (
+          <Button
+            key={filter.label}
+            onClick={() => setSelectedCategory(filter.category)}
+            className={selectedCategory === filter.category && 'bg-size-blue'}
```

Comment:
@samuelrms ref: if your boolean returns false, your className will render it as `className={false}` and that's not ideal. You can use https://github.com/lukeed/clsx to better control your dynamic classes instead!

Reply:
@vaporwavie Thanks for the tip, but I preferred to use tailwind-merge because I'm already using tailwinde, a lib that uses its graphics engine, making it simpler to use.